src/witan/models/dem/ccm/models.clj

```diff
@@ -91,7 +90,7 @@
      :witan/version "1.0.0"
      :witan/type :predicate
      :witan/fn :ccm-core/ccm-loop-pred
-     :witan/params {:last-proj-year 2021}}
+     :witan/params {:last-proj-year 2018}}
```

Comment:
OK. So. Did you read the description above? Essentially 2018 is one of those years which is likely to return either one of the two potential values. When this catalog is used in the test it increases the chance of us having a variable result, so the test is more likely to catch it. Does that make sense?